src/Core/Translation/Storage/Provider/ModuleCatalogueLayersProvider.php

```diff
@@ -234,6 +260,49 @@ private function getUserTranslatedCatalogueFinder(): UserTranslatedCatalogueFind
         return $this->userTranslatedCatalogueFinder;
     }
 
+    /**
+     * @return FileTranslatedCatalogueFinder
+     *
+     * @throws TranslationFilesNotFoundException
+     */
+    private function getBuiltInFileTranslatedCatalogueFinder(): FileTranslatedCatalogueFinder
+    {
+        if (null === $this->builtInFileTranslatedCatalogueFinder) {
+            $this->builtInFileTranslatedCatalogueFinder = new FileTranslatedCatalogueFinder(
+                implode(DIRECTORY_SEPARATOR, [
+                    $this->modulesDirectory,
+                    $this->moduleName,
+                    'translations',
```

Comment:
You need to add the `locale` as well:

`/modules/modulename/translations/<locale>/SomeFile.<locale>.xlf`

Reply:
Oh, my mistake